src/main/resources/jenkins/plugins/gerrit/workflow/Gerrit.groovy

```diff
@@ -67,7 +67,7 @@ class Gerrit implements Serializable {
                     script.scm.getRepositories().each {
                         String name = it.getName()
                         if (it.getName() == rname) {
-                            gerritApiPost(it.getURIs()[0], path, script.USERNAME, script.PASSWORD, jsonPayload)
+                            gerritApiPost(new GerritURI(it.getURIs()[0]).getApiURL(), path, script.USERNAME, script.PASSWORD, jsonPayload)
```

Comment:
Quite surprised we missed that one: there was another PR just merged on that problem that should have worked and had tests associated. Possibly none of them was covering this part of the code though :-(